cider/exec/template/DateTimeTranslator.cpp

```diff
@@ -117,6 +117,53 @@ std::string from_datetrunc_field(const DatetruncField& fieldno) {
 
 }  // namespace
 
+ExtractField ExtractExpr::presto_function_to_extract_field(
```

Comment:
rename to `try_extract_presto_function`?

Reply:
updated